### DISCUSSION TITLE
The StringReplace op doesn't recognise full stops '.'

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#639] [! Welcome - readme first!]
StringReplace doesn't work when trying to replace a full stop.

<img width="1302" alt="Screenshot 2023-12-18 at 10 32 39" src="https://github.com/cables-gl/cables_docs/assets/36418408/8ca50dd8-4d78-41ec-a4ca-049bded402ad">

**Comment by steam0r** [comment]
try `\.`

**Reply by marioepsley** [reply]
Thanks that works :)

**Reply by steam0r** [reply]
for better understanding, it's using regex-syntax: https://dev.cables.gl/op/Ops.String.StringReplace
